src/Compilers/Core/Desktop/CommandLine/CommonCommandLineParser.cs

```diff
@@ -349,8 +355,15 @@ internal static string CheckClientArgsForErrors(IEnumerable<string> args)
                         return CodeAnalysisDesktopResources.KeepAliveIsNotAnInteger;
                     }
                 }
+
+                if (string.Equals(arg, shared, StringComparison.OrdinalIgnoreCase))
```

Comment:
Why do we use `OrdinalIgnoreCase` here and `Ordinal` for `keepAlive` at line 334?